packages/design-system/src/Tag/Tag.types.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+export type Size = "small" | "medium";
```

Comment:
this should be sm, md. Please use Sizes constant from __config__/types.
export type TagSizes = Extract<Sizes, "sm" | "md">